### PR TITLE
Release k6 v0.27.1

### PR DESCRIPTION
**Pull request by na--**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -68,7 +68,7 @@ brew install k6
 
 ### Windows
 
-You can manually download and install the [official `.msi` installation package](https://dl.bintray.com/loadimpact/windows/k6-v0.27.0-amd64.msi) or, if you use the [chocolatey package manager](https://chocolatey.org/), follow [these instructions](https://bintray.com/repo/buildSettings?repoPath=%2Floadimpact%2Fchoco) to set up the k6 repository.
+You can manually download and install the [official `.msi` installation package](https://dl.bintray.com/loadimpact/windows/k6-v0.27.1-amd64.msi) or, if you use the [chocolatey package manager](https://chocolatey.org/), follow [these instructions](https://bintray.com/repo/buildSettings?repoPath=%2Floadimpact%2Fchoco) to set up the k6 repository.
```
Comment (MStoykov) [inline README.md:71]: I am really wondering if this whole section shouldn't just link to the docs?!?

File: release notes/v0.27.1.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+### Other minor fixes and changes:
+
+- The Go version k6 is compiled with was updated to 1.14.6, to incorporate the [latest Go fixes](https://github.com/golang/go/issues?q=milestone%3AGo1.14.6+label%3ACherryPickApproved) ([#1563](https://github.com/loadimpact/k6/pull/1563)).
+- If the [`throw` option](https://k6.io/docs/using-k6/options#throw) is enabled, warnings for failed HTTP requests will no longer be logged to the console ([#1199](https://github.com/loadimpact/k6/issues/1199)).
+- Metric sample packets sent to the cloud with `k6 run --out cloud` can now be sent in parallel via the new `K6_CLOUD_METRIC_PUSH_CONCURRENCY` option, with a default value of `1` ([#1569](https://github.com/loadimpact/k6/pull/1569)).
+- The `gracefulRampDown` VU requirement calculations for the `ramping-vus` executor were greatly optimized for large test runs ([#1567](https://github.com/loadimpact/k6/pull/1567)).
+- Fixed a rare bug where `dropped_iterations` woudln't be emitted by the `per-vu-iterations` executor on time due to a race ([#1357](https://github.com/loadimpact/k6/issues/1357)).
```
Comment (MStoykov) [inline release notes/v0.27.1.md:15]:
```suggestion
- Fixed a rare bug where `dropped_iterations` wouldn't be emitted by the `per-vu-iterations` executor on time due to a race ([#1357](https://github.com/loadimpact/k6/issues/1357)).
```

File: README.md (hunk 1 above)
Comment (na--) [reply README.md:71]: maybe, but not as a part of this PR, add it to https://github.com/loadimpact/k6/issues/1556

File: release notes/v0.27.1.md (hunk 2 above)
Comment (na--) [reply release notes/v0.27.1.md]: Fixed by https://github.com/loadimpact/k6/compare/7bcfb953f09cb905a8eebefad6fbfead7eb03034..15800312f9d98bfcb1d5547b6743701caaeb3a48

**Pull request merged**